src/components/image.js

```diff
@@ -5,11 +5,14 @@ const Image = styled('img')(
   compose(
     layout,
     position
-  ),
-  css`
-    max-height: 100%;
-    max-width: 100%;
-  `
+  )
 );
 
-export { Image };
+const FullSizeImage = styled(Image);
+
+FullSizeImage.defaultProps = {
+  maxWidth: '100%',
+  maxHeight: '100%'
```

Comment:
These props broke regular sized images, but fit the requirement for the full sized ones. Break it out into two components since a one-size-fits-all didn't work.